spec/lib/annotate/annotate_routes_spec.rb

```diff
@@ -333,21 +335,15 @@
         end
 
         it 'annotates without an empty line' do
-          expect(File).to receive(:read).with(ROUTE_FILE).and_return(route_file_content)
           expect(mock_file).to receive(:puts).with(expected_result)
+          expect(AnnotateRoutes).to receive(:puts).with(MESSAGE_ANNOTATED)
```

Comment:
Same with `puts`, I didn't know that the object receiving the message is whatever object that it's being called in.